lib/utils/import/googlekeep/index.ts

```diff
@@ -0,0 +1,101 @@
+import { EventEmitter } from 'events';
+import { endsWith, isEmpty, get, has } from 'lodash';
+import JSZip from 'jszip';
+
+import CoreImporter from '../';
+
+let fs = null;
+const isElectron = has(window, 'process.type');
+if (isElectron) {
+  fs = __non_webpack_require__('fs'); // eslint-disable-line no-undef
+}
+
+class GoogleKeepImporter extends EventEmitter {
+  constructor({ noteBucket, tagBucket, options }) {
+    super();
+    this.noteBucket = noteBucket;
+    this.tagBucket = tagBucket;
+    this.options = options;
+  }
+
+  importNotes = filesArray => {
+    if (isEmpty(filesArray)) {
+      this.emit('status', 'error', 'No file to import.');
+      return;
+    }
+
+    const coreImporter = new CoreImporter({
+      noteBucket: this.noteBucket,
+      tagBucket: this.tagBucket,
+    });
+
+    let importedNoteCount = 0;
+
+    const importJsonString = jsonString => {
+      const importedNote = JSON.parse(jsonString);
+
+      const title = importedNote.title;
+      let textContent = title ? title + '\n\n' : '';
+      textContent += get(importedNote, 'textContent', '');
```

Comment:
this is a minor style issue, but we don't need to mutate this variable when we can instead set it in one go

```ts
const title = importedNote.title;
const importedContent = importedNote.textContent ?? '';
const textContent = title ? `${ title }\n\n${ importedContent }`	: importedContent;
```

Reply:
I removed the mutable variable in `bd9425b`, do you think the code is readable?

Side note: is the `??` operator widely enough supported? With a quick glance, I couldn't find it used elsewhere in this repo.